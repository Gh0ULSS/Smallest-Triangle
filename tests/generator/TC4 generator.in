-N=7 -mindist=2 -rseed=34