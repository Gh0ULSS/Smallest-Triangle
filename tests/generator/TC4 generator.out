2.89, 8.57
34.33, 39.86
38.22, -13.53
44.88, -7.59
-19.38, -40.55
1.17, -34.52
-22.58, 11.06
